### ISSUE TITLE
Impact crashing with other mods

### ISSUE THREAD

**Issue by buscook** [ImpactDevelopment/ImpactIssues#3397]
whenever i try loading in a mod like lambda or autobot the game just crashed on startup completely, i don't have experience looking at crash filed so can any of you help me?

**Comment by zzczzczzcLi** [comment]
​If there have any same hack mod with Impact it will be crashed.Why you do that？But I have met this problem too.The mod DragonCore（a mod edit the attck UI and the main UI just like backpack UI）is crashed with Impact，if there is Impact，MC will crash when loading the DragonCore

**Comment by XiYang20040129** [comment]
yep,the mod DragonCore is crashed with Impact,i dont how to deal it
